### DISCUSSION TITLE
这个报错是什么问题？请问如何解决？

### DISCUSSION THREAD

**Discussion by PhioosJ** [messense/aliyundrive-webdav#481] [Q&A]
m  [2maliyundrive_webdav [0m [2m: [0m failed to check for update: NetworkError: api request failed with status: 401 - for: "https://api.github.com/repos/messense/aliyundrive-webdav/releases/latest"

华硕AC86U 梅林386.5 

aliyundrive-webdav 版本： [aliyundrivewebdav-merlin-arm384-aarch64.tar.gz] (1.6.0)

![微信图片_20220703051450](https://user-images.githubusercontent.com/100673855/177016342-43636ece-530e-425d-b733-920c72fab872.jpg)

**Comment by errpt** [comment]
同样的问题。
